### DISCUSSION TITLE
How can you target h3 and h2 with the same styling?

### DISCUSSION THREAD

**Discussion by montasim** [STRINGLABITSOLUTIONS/Bhojon#210] [Q&A] [answered]
In short.

**Comment by hongducdev** [comment, marked as answer]
There are CSS classes attached to the ezTOC list, example class="ez-toc-heading-level-4".

These do not map directly to h1, h2, h3 …and will be different based on the eligible headings in the post content. But, if you are consistent in using the same headings for TOC items you could use those CSS classes for your target styling.
